packages/schematics/angular/collection.json

```diff
@@ -23,10 +23,10 @@
       "schema": "./application/schema.json",
       "description": "Create an Angular application."
     },
-    "e2e": {
```

Comment:
Note: this is already private as it’s marked as hidden

Reply:
Ah thanks for pointing that out. I think it's still worth renaming this just to identify usage on it which needs to be removed. When we delete Protractor fully, that will likely include this schematic too.